__tests__/app.test.js

```diff
@@ -196,8 +197,52 @@ describe("GET /api/articles",()=>{
             })
         })
     })
+    describe("Articles topic query tests",()=>{
+        it("Should return the articles that have the topic (mitch)",()=>{
+            return request(app)
+            .get("/api/articles?topic=mitch")
+            .expect(200)
+            .then(({body})=>{
+                body.articles.forEach(article => {
+                    expect(article.topic).toBe("mitch")
+                });
+            })
+        })
+
+        it("Should return the articles that have the topic (cats)",()=>{
+            return request(app)
+            .get("/api/articles?topic=cats")
+            .expect(200)
+            .then(({body})=>{
+                body.articles.forEach(article => {
+                    expect(article.topic).toBe("cats")
+                });
+            })
+        })
+        it("Should return the articles that have the topic (paper)",()=>{
+            return request(app)
+            .get("/api/articles?topic=paper")
+            .expect(200)
+            .then(({body})=>{
+                body.articles.forEach(article => {
```

Comment:
there are no articles under the topic paper so this test is being skipped. If you add a length test outside of this you will realize this.